publication-commons/src/main/java/no/unit/nva/publication/service/impl/ResourceService.java

```diff
@@ -374,8 +378,26 @@ private boolean thereAreMorePagesToScan(ScanResult scanResult) {
     }
 
     // change this method depending on the current migration needs.
-    private Resource migrateResource(Resource dataEntry) {
-        return dataEntry;
+    private List<Entity> migrateResource(Resource dataEntry) {
+        if (dataEntry.getEntityDescription() == null) {
+            return List.of(dataEntry);
+        }
+
+        if (dataEntry.getEntityDescription().getContributors().isEmpty()) {
+            return List.of(dataEntry);
+        }
```

Comment:
`isNull(dataEntry.getEntityDescription()) || dataEntry.getEntityDescription().getContributors().isEmpty()` should work, no?